### PR TITLE
Rename package to testcafe-browser-tools (closes #95)

### LINKED ISSUE #95: Rename the package to avoid reference to testcafe
\cc @inikulin @AlexanderMoskovkin @VasilyStrelyaev @MargaritaLoseva 
We need to choose a new name for the package, to state that is a library independent from `testcafe` for managing browser processes and windows.

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @inikulin @georgiy-abbasov

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
-  "name": "testcafe-browser-natives",
-  "version": "0.16.1",
-  "description": "A library used by TestCafe for performing platform-dependent actions on browsers.",
-  "homepage": "https://github.com/DevExpress/testcafe-browser-natives",
-  "bugs": "https://github.com/DevExpress/testcafe-browser-natives/issues",
+  "name": "testcafe-browser-tools",
+  "version": "0.1.0",
```
Comment (inikulin) [inline package.json]: I think we can do it 1.0.0 to make it reliable for plugin developers

**Comment by AndreyBelym** [comment]
Version is `1.0.0` now.

**Comment by inikulin** [comment]
lgtm

**Comment by AndreyBelym** [comment]
\cc @AlexanderMoskovkin @georgiy-abbasov

**Comment by georgiy-abbasov** [comment]
lgtm, but i have one minor question. Maybe we should rename namespaces in .NET source files ?

**Comment by AndreyBelym** [comment]
Of course I should.

File: test/playground/views/test-page.mustache [hunk 2]
```diff
@@ -20,7 +20,29 @@
...
+        //else
+        sendXhr({ url: '/confirm-close/{{{ id }}}', sync: false });
+    }
+
+    function sendClientAreaSize (clientAreaSize, callback) {
+        sendXhr({
+            url:  '/update-client-area-size/{{ id }}',
+            data: JSON.stringify(clientAreaSize)
+        }, callback);
+>>>>>>> Stashed changes
```
Comment (AlexanderMoskovkin) [inline test/playground/views/test-page.mustache]: something went wrong here

File: test/playground/views/test-page.mustache [hunk 3]
```diff
@@ -37,6 +59,15 @@
...
     window.addEventListener('resize', updateClientAreaSize);
+<<<<<<< Updated upstream
+=======
+    window.addEventListener('beforeunload', function (e) {
+        sendCloseConfirmation();
+        e.preventDefault();
+        e.returnValue = 'test';
+        return 'test';
+    });
+>>>>>>> Stashed changes
```
Comment (AlexanderMoskovkin) [inline test/playground/views/test-page.mustache]: and here

**Comment by AlexanderMoskovkin** [comment]
Are texts are approved by a techwriter?

lgtm

**Comment by AndreyBelym** [comment]
@VasilyStrelyaev , please check changes in texts

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by georgiy-abbasov** [comment]
lgtm

**Comment by inikulin** [comment]
We need to publish it under the new name, and publish deprecation notice for old package

**Pull request merged**